Public/Src/Cache/ContentStore/Distributed/NuCache/BlobCentralStorage.cs

```diff
@@ -96,15 +106,15 @@ protected override async Task<BoolResult> TryGetFileCoreAsync(OperationContext c
                 AttemptResult result = await TaskUtilities.WithTimeoutAsync(async token =>
                     {
                         var blob = container.GetBlockBlobReference(blobName);
-                        var exists = await blob.ExistsAsync(null, null, token);
+                        var exists = await blob.ExistsAsync(DefaultBlobStorageRequestOptions, null, token);

```

Comment:
I'm not sure that we should use these new default options here... What do you think?